src/rpcblockchain.cpp

```diff
@@ -777,6 +781,16 @@ UniValue getchaintips(const UniValue& params, bool fHelp)
     return res;
 }
 
+UniValue mempoolInfoToJSON()
+{
+    UniValue ret(UniValue::VOBJ);
+    ret.push_back(Pair("size", (int64_t) mempool.size()));
+    ret.push_back(Pair("bytes", (int64_t) mempool.GetTotalTxSize()));
+    ret.push_back(Pair("usage", (int64_t) mempool.DynamicMemoryUsage()));
```

Comment:
This is not called locked, so it is possible that the mempool changes between these calls. (For example, dividing bytes by size could give a slightly inaccurate measure of average transaction bytes.)